Lib/test/test_sax.py

```diff
@@ -254,6 +254,12 @@ def test_make_parser2(self):
         from xml.sax import make_parser
         p = make_parser()
 
+    def test_make_parser3(self):
+        # Testing that make_parser can handle iterables other than a
+        # list.
+        make_parser(('module', ))
+        make_parser({'module'})
+        make_parser({'module': None})
```

Comment:
This should also test a non-collection iterable, e.g. `iter(['module'])`.

Also, make sure to have at least one test using a zero-length `parser_list`, as well as other tests with a `parser_list` with more than one item.

Reply:
Thanks! I hope I have addressed your comments :)